cloud/cloud_file_cache.cc

```diff
@@ -109,12 +120,35 @@ uint64_t CloudEnvImpl::FileCacheGetCharge() {
 // This is not thread-safe and is used only for unit tests.
 //
 uint64_t CloudEnvImpl::FileCacheGetNumItems() {
-  assert(GetCloudEnvOptions().sst_file_cache);
   clear_callback_state();
-  GetCloudEnvOptions().sst_file_cache->get()->ApplyToAllCacheEntries(callback,
+  cloud_env_options.sst_file_cache->ApplyToAllCacheEntries(callback,
                                                                      true);
   return callback_state.size();
 }
 
+// Removes all items for the env from the cache.
+// This is not thread-safe.
+void CloudEnvImpl::FileCachePurge() {
+  if (!cloud_env_options.hasSstFileCache()) {
```

Comment:
What if I start RocksDB with a non-zero file cache size and then set it to zero? This check will fail and you will not purge the existing file cache.